new-log-viewer/src/components/Editor/MonacoInstance/typings.ts

```diff
@@ -7,6 +7,17 @@ import * as monaco from "monaco-editor/esm/vs/editor/editor.api.js";
 import {ACTION_NAME} from "../../../utils/actions";
 
 
+enum TOKEN_NAMES {
```

Comment:
```suggestion
enum TOKEN_NAME {
```
Plural naming is reserved for arrays.